src/Raven.Client/Documents/Operations/Backups/BackMode.cs

```diff
@@ -0,0 +1,7 @@
+namespace Raven.Client.Documents.Operations.Backups;
+
+public enum BackMode
```

Comment:
BackMode? :)

Reply:
maybe lets call it BackupUploadMode? Thoughts?

Reply:
done